runtime/tests/interpreter/entitlements_test.go

```diff
@@ -933,268 +948,6 @@ func TestInterpretEntitledReferenceCasting(t *testing.T) {
 
 }
 
-func TestInterpretCapabilityEntitlements(t *testing.T) {
```

Comment:
See Slack question

Reply:
I'm not sure I follow why this test has been removed in its entirety. If https://dapperlabs.slack.com/archives/CG0B7CJAJ/p1687556719100839 is the Slack comment you are referring to, I don't think these tests assume the existence of an untyped Capability, rather they are testing that Capabilities can be borrowed with supertypes of the Capability's actual type.

Reply:
Added the tests back as runtime tests in 38ca7a102847ed4b5eeed159d632298b8e693bd2